_data/CONTRIBUTORS.yaml

```diff
@@ -518,3 +518,8 @@ Nadja Žlender:
     orcid: 0000-0001-5913-5530
     email: nadja.zlender@mf.uni-lj.si
     affiliation: University of Ljubljana, Medical Faculty, ELIXIR-SI
+Dirk Winkelhardt:
+    git: di-hardt
+    orcid: 0000-0001-8770-2221
+    email: dirk.winkelhardt@rub.de
+    affiliation: Ruhr-Universität Bochum, Medizinische Fakultät, Medzinisches Proteom-Center & ELIXIR Germany, Institut für Bio- and Geosciences (IBG-5), Forschungszentrum Jülich GmbH
```

Comment:
@di-hardt That is a very long affiliation that will render a bit weird on the website. Any chance you could shorten it? 😅

Reply:
Ah yes, sure. Here you go: [fa3dc34](https://github.com/di-hardt/rdmkit-demo/commit/fa3dc3443435d0fe2a3731ddec89ff3ae0e27df3)

Reply:
Danke!